packages/salesforcedx-vscode-core/src/commands/isvdebugging/bootstrapCmd.ts

```diff
@@ -184,7 +184,10 @@ export class IsvDebugBootstrapExecutor extends SfdxCommandletExecutor<{}> {
       )
       .withArg('force:mdapi:retrieve')
       .withFlag('--retrievetargetdir', this.relativeMetdataTempPath)
-      .withFlag('--packagenames', packageNames.join(','))
+      .withFlag(
```

Comment:
These changes seem unrelated to the documentation ones. We should split this into different PRs ?

Reply:
Yes. they should be removed from this PR.

Reply:
Sorry, I thought this was a fresh branch off of develop. This was my mistake.